packages/model-viewer/src/three-components/gltf-instance/correlated-scene-graph.ts

```diff
@@ -119,7 +120,7 @@ export class CorrelatedSceneGraph {
     const originalGLTF = upstreamCorrelatedSceneGraph.gltf;
     const cloneGLTF: GLTF = JSON.parse(JSON.stringify(originalGLTF));
     const cloneThreeObjectMap: ThreeObjectToGLTFElementHandleMap = new Map();
-    const cloneGLTFELementMap: GLTFElementToThreeObjectMap = new Map();
+    const cloneGLTFElementMap: GLTFElementToThreeObjectMap = new Map();
```

Comment:
I'm guessing this rename was unintentional?

Reply:
Typo? Was it supposed to be ELementMap vs ElementMap?

Reply:
oh, oops; I read it backwards. Thanks!